hpOneView/resources/storage/storage_systems.py

```diff
@@ -24,45 +24,18 @@
 
 standard_library.install_aliases()
 
-from hpOneView.resources.resource import ResourceClient
+from hpOneView.resources.resource import Resource
```

Comment:
is it not required to import ResourceHelper class here?

Reply:
Not required here as we are not using any heloper methods here. Resource class is the base class for all the resources